server/lib/manifest.py

```diff
@@ -0,0 +1,389 @@
+import os
+
+from girder.utility.model_importer import ModelImporter
+
+
+class Manifest:
+    """
+    Class that represents the manifest file.
+    Methods that add information to the manifest file have the form
+    add_<someProperty>
+    while methods that create chunks of the manifest have the form
+    create<someProperty>
+    """
+
+    def __init__(self, tale_license, item_ids=None):
```

Comment:
Ultimately we always want to create a manifest for a Tale. Would it make sense to pass that Tale object here and store it internally?

Reply:
We use it enough where it would be practical; do you think I should do the same with `user`?